deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/local_actions.py

```diff
@@ -94,5 +94,5 @@ def qa_ctl_docker_run(config_file, qa_branch, debug_level, topic):
     run_local_command_with_output(f"cd {docker_image_path} && docker build -q -t {docker_image_name} .")
 
     LOGGER.info(f"Running the Linux container for {topic}")
-    run_local_command(f"docker run --rm -v {os.path.join(gettempdir(), 'qa_ctl')}:/qa_ctl {docker_image_name} "
+    run_local_command(f"docker run --rm -v {os.path.join(gettempdir(), 'wazuh_qa_ctl')}:/qa_ctl {docker_image_name} "
```

Comment:
Is this line working properly?

Note that you have renamed the docker directory, and here you are mounting the directory `gettempdir()/wazuh_qactl` in `/qa_ctl` of the docker, which a priori should no longer exist

Reply:
This error has been fixed in the commit [dfca4e8](https://github.com/wazuh/wazuh-qa/commit/dfca4e82096968bd5d4133fb87cc70600ea4e964)